src/coreModules/commonUi/translations.json

```diff
@@ -33,6 +33,18 @@
     "dynamic": {
       "en": "dynamic",
       "sv": "dynamisk"
+    },
+    "overview": {
+      "en": "overview",
+      "sv": "översikt"
+    },
+    "versions": {
+      "en": "versions",
+      "sv": "versioner"
+    },
+    "models": {
```

Comment:
do we aim to keep translations in alphabetic order?

Reply:
hmm lets talk about that

Reply:
ok